README.md

```diff
@@ -149,16 +149,17 @@ Please be aware, specifying the scheme value as a part of the `runArguments` par
 
 To debug a project created using Expo or the `create-react-native-app` task, you can use embedded support for Expo.
 
-Your environment must meet the following prerequisites:
+Prepare your environment using an [introduction](https://reactnative.dev/docs/environment-setup). Pay attention to:
 
 - Install the [Expo app](https://getexponent.com/) on the target device or emulator
-- Ensure that the `react-native-cli` is installed globally (`npm install -g react-native-cli`)
+- Ensure that the `Android SDK` is installed on your computer (You may install it with [Android Studio](https://developer.android.com/studio))
```

Comment:
It is better to point out how exactly it should be installed. For example after installation of Android SDK some of `PATH` environment variables may absent and this might lead to errors as in #1230

Reply:
Maybe suggest too the option to run `npx react-native doctor` to validate that everything is installed and configured correctly?
Soon-ish [doctor will install Android Studio and SDK on Windows automatically 😊](https://github.com/react-native-community/cli/pull/1109)

Reply:
Sounds good! It may help to find gaps in Android SDK configuration.